sdk/storage/storage-file/src/DirectoryClient.ts

```diff
@@ -1140,6 +1139,46 @@ export class DirectoryClient extends StorageClient {
     }
   }
 
+  /**
+   * Force close all handles for a directory.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles
+   *
+   * @param {DirectoryForceCloseHandlesSegmentOptions} [options={}]
+   * @returns {Promise<number>}
+   * @memberof DirectoryClient
+   */
+  public async forceCloseAllHandles(
+    options: DirectoryForceCloseHandlesSegmentOptions = {}
```

Comment:
We also want a boolean `recursive` parameter.  Is it supported by `DirectoryForceCloseHandlesSegmentOptions` or do we have to navigate the directory structure by ourselves?

Reply:
`recursive` parameter is present in `DirectoryForceCloseHandlesSegmentOptions`.

Reply:
Good.